backend/battles/battle.py

```diff
@@ -0,0 +1,45 @@
+from .email import send_battle_result  # noqa
+
+
+def win_attack(pokemon_1, pokemon_2):  # pokemon_1 always attacks and pokemon_2 always defends
+    if pokemon_1.attack > pokemon_2.defense:
+        return True
+    return False
+
+
+def highest_hp_pokemon(pokemon_1, pokemon_2):
+    if pokemon_1.hp > pokemon_2.hp:
+        return pokemon_1
+    return pokemon_2
```

Comment:
Same thing here.

```suggestion
def highest_hp_pokemon(pokemon_1, pokemon_2):
    return pokemon_1.hp > pokemon_2.hp
```

Also, same thing about the name. Maybe `get_highest_hp_pokemon` or `get_pokemon_with_highest_hp`. It's okay to have lengthy function names as long as they're descriptive.